blog-posts/sls-mentor/release/article.md

```diff
@@ -0,0 +1,215 @@
+
+The only task left to pass level 1 is to enable server-side encryption on my blog-articles bucket Using the AWS CDK, I need to specify in the Bucket construct I used to provision the bucket, that I want to enable encryption. I chose simple `S3_MANAGED` encryption to start.
+
+```typescript
+// Using the CDK
+
+const exampleBucket = new Bucket(this, 'BlogArticles', {
+  bucketName: 'sls-mentor-blog-articles',
+  encryption: BucketEncryption.S3_MANAGED, // Enable encryption
+});
+```
+
+Last level 1 run of sls-mentor! Everything is green! See, it was not too hard.
+
+![sls-mentor results](./assets/results-3.png 'sls-mentor results')
+
+Level 1 is complete, there are 4 other levels to conquer! To end this article, I will have a quick look at level 2, where I will teach you some advanced sls-mentor tricks...
+
+### A look at sls-mentor level 2 + advanced sls-mentor configuration
+
+Let's take a look at the level 2 results:
+
+![sls-mentor results](./assets/results-4.png 'sls-mentor results')
+
+It's not that bad! Level 2 includes level 1 so half of the work was already done, but some rules like "Lambda - Under max memory" are already passing everywhere.
+
+Here, I want to focus on the rule "S3 - Use intelligent tiering". This rule enforces the usage of intelligent tiering in my buckets, it allows infrequently accessed objects to be stored in lower tiers, thus reducing their carbon footprint and cost.
+
+On my application, I use CDK to deploy my bucket, so I will need to add a new lifecycle rule to my bucket definition, transitioning items to IntelligentTiering on their first day.
+
+```typescript
+// Using the CDK
+
+const exampleBucket = new Bucket(this, 'BlogArticles', {
+  bucketName: 'sls-mentor-blog-articles',
+  encryption: BucketEncryption.S3_MANAGED,
+  lifeCycleRules: [
+    // add a lifecycle rule
+    {
+      id: 'auto-intelligent-tiering',
+      enabled: true,
+      transitions: [
+        {
+          // New objects transition to intelligent tiering on day 0
+          storageClass: StorageClass.INTELLIGENT_TIERING,
+          transitionAfter: Duration.days(0),
+        },
+      ],
+    },
+  ],
+});
+```
+
+But there's a catch, on the dashboard, sls-mentor tells me that 2 buckets failed the check! In fact, the second one is my serverless deployment bucket, use during deployment by the Serverless framework. I do not have access to its configuration in my code.
+
+We anticipated this issue: using a simple `sls-mentor.json` configuration file, I can define resources that won't be checked. In the case of my serverless deployment bucket, I just need to write the following lines in the file:
+
+```json
+{
+  "rules": {
+    "useIntelligentTiering": {
+      "ignoredResources": ["serverlessdeployment"]
+    }
+  }
+}
+```
+
+The ignoredResources array is composed of regex, so I just need to specify `serverlessdeployement` to avoid checking all resources with ARN matching this pattern.
```

Comment:
"serverlessdeployment" does not quite look like a regexp. Maybe worth adding how the matching works? (contains substring?)

Reply:
ok let's go with ^.* serverlessdeployment.*$